js/packages/openinference-instrumentation-langchain/README.md

```diff
@@ -10,18 +10,23 @@ npm install --save @arizeai/openinference-instrumentation-langchain
 
 ## Usage
 
-To load the Langchain instrumentation, specify it in the registerInstrumentations call along with any additional instrumentation you wish to enable.
+To load the Langchain instrumentation, manually instrument the `@langchain/core/callbacks/manager` module. The callbacks manager must be manually instrumented due to the non-traditional module structure in `@langchain/core`. Additional instrumentations can be registered as usual in the registerInstrumentations function.
 
 ```typescript
 import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
 import { LangChainInstrumentation } from "@arizeai/openinference-instrumentation-langchain";
 import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import * as CallbackManagerModule from "@langchain/core/callbacks/manager";
 
 const provider = new NodeTracerProvider();
 provider.register();
 
+const lcInstrumentation = new LangChainInstrumentation();
+// LangChain must be manually instrumented as it doesn't have a traditional module structure
+lcInstrumentation.manuallyInstrument(CallbackManagerModule);
+
 registerInstrumentations({
```

Comment:
You can mix this then??

Reply:
ahh yeah good point, i wasn't sure if it should still be passed in for unwrapping, not sure it would work anyways though